.travis.yml

```diff
@@ -47,18 +48,22 @@ matrix:
     - php: hhvm
 
 before_install:
-  - if [[ $TRAVIS_PHP_VERSION != "hhvm" && $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini ; fi
+  - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
+  - travis_retry composer self-update
```

Comment:
This line is redundant. Travis update composer automatically, please see:
https://travis-ci.org/zfcampus/zf-asset-manager/jobs/296348988#L473
https://travis-ci.org/zfcampus/zf-asset-manager/jobs/296348988#L478